src/volta-shim.rs

```diff
@@ -1,35 +1,9 @@
-mod common;
+use std::process::{Command, ExitCode};
 
-use common::{ensure_layout, Error, IntoResult};
-use volta_core::error::{report_error, ExitCode};
-use volta_core::log::{LogContext, LogVerbosity, Logger};
-use volta_core::run::execute_shim;
-use volta_core::session::{ActivityKind, Session};
-use volta_core::signal::setup_signal_handler;
-
-pub fn main() {
-    Logger::init(LogContext::Shim, LogVerbosity::Default)
-        .expect("Only a single Logger should be initialized");
-    setup_signal_handler();
-
-    let mut session = Session::init();
-    session.add_event_start(ActivityKind::Tool);
-
-    let result = ensure_layout().and_then(|()| execute_shim(&mut session).into_result());
+pub fn main() -> ExitCode {
```

Comment:
Could add some info to the description about the flow that execution goes through on both Windows and Unix after this change? I think I understand it, but I want to make sure I'm tracking it correctly.

For example, the status quo before this change is:

### Unix
node -(symlink)-> volta_shim -> (resolve version) -> `node`

### Windows
node -(symlink)-> volta_shim -> (resolve version) -> `cmd.exe /C node`

Reply:
On Windows would be:

node -(runner)-> volta_shim -> (resolve version) -> cmd.exe /C node